python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/dot1x.schema.yml

```
@@ -18,6 +18,9 @@ keys:
         type: bool
       dynamic_authorization:
         type: bool
+      statistics_packets_dropped:
+        type: bool
+        description: Enable the 802.1X port authentication dropped data packet statistics.
```

Should it be plural like the key?
```suggestion
        description: Enable the 802.1X port authentication dropped data packets statistics.
```

---

nope this is the help from EOS!